cmds/core/grep/grep.go

```diff
@@ -83,15 +76,14 @@ func main() {
 
 // cmd contains the actually business logic of grep
 type cmd struct {
-	stdin      io.ReadCloser
-	stdout     *bufio.Writer
-	stderr     io.Writer
-	args       []string
-	exprB      []byte
+	stdin  io.ReadCloser
+	stdout *bufio.Writer
+	stderr io.Writer
+	args   []string
+	exprB  []byte
```

Comment:
Since we're renaming things, I think expr is find, I don't think it needs to be exprB? in general Go style is to avoid annotating variable names with types -- it's messy if the type changes later. Unless the B has some other meaning? in which case it's a bit confusing.

Reply:
I think exprB is here because previous PR switched comparison logic to bytes and to not to pay conversion cost additional variable exprB was introduced. I actually don't see big in doing that, so I removed it back to strings and remove exprB at all. Maybe we can gain some performance by doing bytes over strings, but we don't play in ripgrep league anyway.